inst/assump/costIntangRen.csv

```diff
@@ -1,9 +1,16 @@
 .chunk;bs;hs;bsr;hsr;vin;reg;loc;typ;inc;ttot;value
-0;;;;sobo;;;;;;;100
-0;;;;reel;;;;;;;-10
-0;;;;libo;;;;;;;-10
-1;;;;biom;;;;;;2000;20
-1;;;;biom;;;;;;2030;0
-1;;;;ehp1;;;;;;2000;-20
-1;;;;ehp1;;;;;;2030;-10
-
+0;;;;"0";;;;;;;-20
+0;;;;reel;;;;;;;-20
+0;;;;libo;;;;;;;-22
+1;;;;biom;;;;;;2010;20
+1;;;;biom;;;;;;2040;0
+1;;;;ehp1;;;;;;2005;-30
+1;;;;ehp1;;;;;;2040;10
+1;;;;dihe;;;;;;2010;10
+1;;;;dihe;;;;;;2030;25
+1;;;;gabo;;;;;;2030;-18
+1;;;;gabo;;;;;;2050;-25
+2;;reel;;;;;;;;;50
+3;;reel;;"0";;;;;;;-20
+3;;reel;;reel;;;;;;;-20
+#4;;reel;;gabo;;;;;;;10000
```

Comment:
What is the reason for this commented line? I guess we would aim to have no commented code or similar anywhere,

Reply:
I agree. Thanks for spotting.